### ISSUE TITLE
Colour contrast issues with tabs on tool page

### ISSUE THREAD

**Issue by nickytonline** [open-sauced/app#2358]
Colour contrast issues with tabs on tool page

The current text colours for non-active tabs fails colour contrast checks.

![CleanShot 2023-12-15 at 17 34 20](https://github.com/open-sauced/app/assets/833231/b35baf54-cf79-4ff0-84c9-212669a771c6)
